internal/proxy/clientgo_cached_proxy_resolver_test.go

```diff
@@ -99,6 +100,29 @@ func Test_FetchCustomEntities(t *testing.T) {
 	}
 }
 
+func TestIsReady(t *testing.T) {
+	fakePostgresKong := sendconfig.Kong{InMemory: false}
+	fakeDblessKong := sendconfig.Kong{InMemory: true}
+	postgresProxy := clientgoCachedProxyResolver{
+		kongConfig: fakePostgresKong,
+		dbmode:     "postgres",
+	}
+	dblessProxy := clientgoCachedProxyResolver{
+		kongConfig: fakeDblessKong,
+		dbmode:     "off",
+	}
+
+	t.Log("checking initial readiness state")
+	assert.Equal(t, true, postgresProxy.IsReady())
+	assert.Equal(t, false, dblessProxy.IsReady())
+
+	t.Log("checking readiness after lastConfigSHA is set")
+	postgresProxy.lastConfigSHA = []byte{1, 2, 3, 4}
+	dblessProxy.lastConfigSHA = []byte{1, 2, 3, 4}
+	assert.Equal(t, postgresProxy.IsReady(), true)
+	assert.Equal(t, dblessProxy.IsReady(), true)
```

Comment:
nit:
use `assert.True`, `assert.False` instead of `assert.Equal` for comparing booleans to literal values

Reply:
Part of 93387e8b.